tools/src/main/scala/scala/scalanative/codegen/Generate.scala

```diff
@@ -138,6 +138,8 @@ object Generate {
         Next.Unwind(exc, Next.Label(handler, Seq(exc)))
       }
 
+      lazy val clinitCalls = genClinitCalls(unwind)
```

Comment:
A `lazy val` that calls a thing with by-name argument that has side effects is dangerous. Since `clinitCalls` is always called exactly once anyway, it would be easier to reason about by simply inlining `genClinitCalls(unwin)` at the only place where `clinitCalls` is used.